README.md

```diff
@@ -77,25 +77,34 @@ The above command alone will have `coredns` binary generated.
 
 ## Examples
 
-When starting CoreDNS without any configuration, it loads the
+To start CoreDNS, use `./coredns` when you are in the coredns directory. When starting CoreDNS without any configuration, it loads the
```

Comment:
If this is in windows environment without posix like shell, `./coredns` will not work as Windows Cmd environment does not recognize `./`. Also, if binary coredns is under the directory specified in `PATH` environment, a `coredns` would be enough. Let's remove this change as it is too restricted to the environment setup.

Reply:
Sure, I'll make the necessary change.

Reply:
I've changed all environment-specific documentation.